Backend/src/main/java/ca/mcgill/cooperator/service/CoopService.java

```diff
@@ -78,6 +78,15 @@ public Coop getCoopById(int id) {
         return c;
     }
 
+    @Transactional
```

Comment:
I just moved these so that all the get methods were grouped together, I didn't actually change anything